actors/market/tests/harness.rs

```diff
@@ -166,6 +166,34 @@ pub fn expect_provider_control_address(
     expect_get_control_addresses(rt, provider, owner, worker, vec![])
 }
 
+pub fn expect_get_is_control_addresses(
```

Comment:
This method has the wrong name. I think you should inline it into `expect_provider_is_control_address`, which is otherwise doing nothign.